src/main/java/me/zeroeightsix/kami/module/modules/movement/ElytraFlight.kt

```diff
@@ -182,38 +202,134 @@ class ElytraFlight : Module() {
         } else elytraDurability = 0
 
         /* Holds player in the air if run out of durability */
-        if (!mc.player.onGround && outOfDurability && elytraDurability <= 1) {
-            event.cancel()
-            mc.player.setVelocity(0.0, -0.01, 0.0)
+        if (!mc.player.onGround && elytraDurability <= 1 && outOfDurability) {
+            holdPlayer(event)
         } else if (outOfDurability) outOfDurability = false /* Reset if players is on ground or replace with a new elytra */
```

Comment:
outOfDurability is also never assigned to something other then false

Reply:
Check line 195

Reply:
uhhhhh what
![dominikaaaa org-2020-07-08-20-28-38](https://user-images.githubusercontent.com/17222512/86983360-b0c25f80-c159-11ea-983d-1ef14087fa8a.png)

Reply:
The new line 195 not the old one